tests/python_package_test/test_plotting.py

```diff
@@ -257,7 +262,7 @@ def test_tree_with_categories_below_max_category_values():
 
 
 @pytest.mark.skipif(not GRAPHVIZ_INSTALLED, reason="graphviz is not installed")
-def test_tree_with_categories_above_max_category_values():
+def test_tree_with_categories_above_max_category_values(tmp_path: Path):
```

Comment:
```suggestion
def test_tree_with_categories_above_max_category_values(tmp_path):
```

Please revert this. We prefer not to use type hints on `pytest` test-case functions in this project.